pkg/webhooks/policymutation.go

```diff
@@ -29,25 +31,20 @@ func (ws *WebhookServer) policyMutation(request *v1beta1.AdmissionRequest) *v1be
 	}
 
 	if request.Operation == v1beta1.Update {
-		if err := json.Unmarshal(request.OldObject.Raw, &oldPolicy); err != nil {
-			logger.Error(err, "failed to unmarshall old policy admission request")
-			return &v1beta1.AdmissionResponse{
-				Allowed: true,
-				Result: &metav1.Status{
-					Message: fmt.Sprintf("failed to default value, check kyverno controller logs for details: %v", err),
-				},
-			}
-		}
-
-		if isStatusUpdate(oldPolicy, policy) {
-			logger.V(3).Info("skip policy mutation on status update")
-			return &v1beta1.AdmissionResponse{Allowed: true}
+		admissionResponse := hasPolicyChanged(policy, request.OldObject.Raw, logger)
+		if admissionResponse != nil {
+			logger.V(4).Info("skip policy mutation on status update")
+			return admissionResponse
 		}
 	}
 
+	startTime := time.Now()
+	logger.V(3).Info("start mutating policy")
```

Comment:
should this be, "start policy change validation" and "finished start policy change validation" instead?